pages/projects/life-journeys.js

```diff
@@ -71,21 +71,17 @@ export default function LifeJourneys(props) {
         <section className="layout-container relative mb-10">
           <h1
             id="pageMainTitle"
-            className="mb-8 text-h1l font-bold flex-wrap"
+            className="mb-8 text-h1l flex-wrap"
```

Comment:
I was gonna say, we don't want to use "strong" in headings, but it looks like they are still bold, so all good.